README.md

```diff
@@ -263,16 +263,16 @@ docker run --gpus all -it --rm  -v $(pwd):/home/app/mava -w /home/app/mava insta
     * Install core dependencies:
 
     ```bash
-    pip install id-mava[tf,reverb,launchpad]
```

Comment:
I'm not quite convinced by removing `launchpad` as a dependency, since we use it directly. Unlikely to break anything though, since like you said, it's a core dependency of `acme`. So I don't feel very strongly about keeping it in

Reply:
I think the issue with having a separate dependency while it is a core dependency on acme is managing the version. Acme will have its own required version of lp, if ours doesn't match then the install fails due to a conflict. Becomes tricky to always manage the correct version.

Reply:
If you just don't give a specific version in our requirements for `launchpad`, then it will use the `acme` version, I think

Reply:
Yeah that is true, but then I don't see the benefit of having it as a separate dependency. If acme removes it as a core dependency, then we can maybe have it as a separate dependency. What do you think?

Reply:
Sounds good :)